lib/tapioca/commands/dsl.rb

```diff
@@ -115,8 +115,13 @@ def execute
 
           if @auto_strictness
             say("")
-            update_gem_rbis_strictnesses([], gem_dir: @gem_dir, dsl_dir: @outpath.to_s)
-            say("")
+            validate_rbi_files(
+              command: default_command(:dsl, @requested_constants.join(" ")),
+              gem_dir: @gem_dir,
+              dsl_dir: @outpath.to_s,
+              auto_strictness: @auto_strictness,
+              compilers: Runtime::Reflection.descendants_of(Tapioca::Dsl::Compiler)
```

Comment:
I think, we should be able to ask the pipeline exactly which compilers were enabled instead:
```suggestion
              compilers: pipeline.compilers
```